app/uk/gov/hmrc/agentsexternalstubs/controllers/DesStubController.scala

```diff
@@ -292,6 +292,10 @@ class DesStubController @Inject()(
               businessPartnerRecordsService
                 .getBusinessPartnerRecord(Utr(utr), session.planetId)
                 .flatMap(getOrCreateBusinessPartnerRecord(payload, idType, idNumber, session.planetId)(record => true))
+            case ("urn", urn) =>
```

Comment:
do we need this? doesn't it belong to POST          /registration/individual/:idType/:idNumber   ?

Reply:
you maybe right, I will have a look